.circleci/config.yml

```diff
@@ -141,7 +125,6 @@ workflows:
           requires:
             - lint
             - unit
-            - typecheck
```

Comment:
did we miss that in the example pkg? if yes, can you please add it there too?

Reply:
It's removed because this is plain javascript

Reply:
ah removed - I'm sorry, I mixed it up with an addition 🙈